.gitignore

```diff
@@ -1,2 +1,4 @@
 *.env*
 __pycache__
+.vscode/
+data_test/.timetrackdb*
```

Comment:
if you will have a sample for tests, (for python projects) you normally add it to`/tests/data`